src/components/RenderArticles/RenderArticles.jsx

```diff
@@ -1,22 +1,37 @@
 import { useState, useEffect } from 'react'
-import { getArticles } from '../../utils/utils'
+import apiCall from '../../hooks/apiCall'
 import { ArticleItem } from './components/ArticleItem/ArticleItem'
 import { LoadMoreBtn } from '../LoadMoreBtn/LoadMoreBtn'
 import './RenderArticles.css'
 
-export const RenderArticles = () => {
+export const RenderArticles = ({queries}) => {
     const [articles, setArticles] = useState([])
     const [isLoading, setIsLoading] = useState(true)
     const [pageNumber, setPageNumber] = useState(1)
     const [totalArticles, setTotalArticles] = useState(-1)
 
     useEffect(() => {
+        setArticles([])
```

Comment:
not entirely sure why you have this useEffect here? you are setting the articles to an empty array when the queries changes but this is being overridden by the second useEffect which is also listening out for any changes in queries...